.github/workflows/ci.yml

```diff
@@ -7,29 +7,30 @@ on:
     branches-ignore:
       - gh-pages
 
+permissions:
+  contents: read
+
 jobs:
   build:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
     defaults:
       run:
         working-directory: standards-catalogue
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
 
-      - uses: ruby/setup-ruby@v1
+      - uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
```

Comment:
Worth updating this in line with https://github.com/co-cddo/api-catalogue/pull/321 ?